src/main/java/org/cryptomator/ui/unlock/UnlockWorkflow.java

```diff
@@ -115,6 +133,8 @@ protected void failed() {
 		Throwable throwable = super.getException();
 		if(throwable instanceof IllegalMountPointException impe) {
 			handleIllegalMountPointError(impe);
+		} else if (throwable instanceof FuseRestartRequiredException fRRE) {
```

Comment:
No need for weird acronyms (fix the other if-branches as well, please)

```suggestion
		} else if (throwable instanceof FuseRestartRequiredException e) {
```

Reply:
> No need for weird acronyms (fix the other if-branches as well, please)

while this specific change can be reverted, the other acronyms are not part of this PR.